packages/react/error-boundary/src/useErrorBoundary.en.md

```diff
@@ -0,0 +1,19 @@
+# useErrorBoundary
+
+Hook used to deliver the error to the nearest `ErrorBoundary`.
+
+React's [`ErrorBoundary`](https://reactjs.org/docs/error-boundaries.html) component only catches an error occurred on render or in `useEffect` callback.
+So `useErrorBoundary` returns a function to deliver any error occurred in other place to `ErrorBoundary`.
```

Comment:
Sorry. I fixed it more clear:
```markdown
React's [error boundary](https://reactjs.org/docs/error-boundaries.html) component cannot catch an error occurred in event handlers, asynchronous code like `setTimeout`, etc.

`useErrorBoundary` is useful to deliver an error to the nearest error boundary in anywhere.
```

Reply:
[fix: useErrorBoundary docs more clear](https://github.com/toss/slash/pull/102/commits/7f58b2cc3022504db5efb7f1b5508fa36da12f24)
[7f58b2c](https://github.com/toss/slash/pull/102/commits/7f58b2cc3022504db5efb7f1b5508fa36da12f24)

fixed! 
thanks! I got it